tools/myst/host/exec.c

```diff
@@ -403,8 +406,75 @@ int exec_action(int argc, const char* argv[], const char* envp[])
 
     unlink(archive_path);
 
+    oe_sgx_enclave_setting_config_data config_data_setting_optional = {
+        {0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15,
+         0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15,
+         0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15,
+         0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15},
+        2,
+        true /* ignore_if_unsupported */};
+    {
+        void* buffer;
+        size_t buffer_size;
+        if (myst_load_file(roothashes[0], (void**)&buffer, &buffer_size) == 0)
+        {
+            myst_sha256_t sha256;
+            myst_sha256(&sha256, buffer, buffer_size);
+            uint8_t* tmp = (uint8_t*)buffer;
+            for (int i = 0; i < 32; i++)
+            {
+                char tmp_buf[3];
+                tmp_buf[2] = 0;
+                for (int j = 0; j < 2; j++)
+                {
+                    tmp_buf[j] = tmp[i * 2 + j];
+                }
+                uint8_t ui1;
+                ui1 = (uint8_t)strtol(tmp_buf, NULL, 16);
+                // printf("%3x", ui1);
+                // if ((i + 1) % 8 == 0) {
+                //     printf("\n");
+                // }
+                config_data_setting_optional.config_id[i] = ui1;
+            }
+        }
+    }
+
+    if (commandline_config)
+    {
+        void* buffer;
+        size_t buffer_size;
+        if (myst_load_file(commandline_config, (void**)&buffer, &buffer_size) ==
+            0)
+        {
+            myst_sha256_t sha256;
+            myst_sha256(&sha256, buffer, buffer_size);
```

Comment:
Do we need to concatenate two hashes or having just one (chaining two together)?